### DISCUSSION TITLE
Problema stato luci tra impianto domotico e hassio

### DISCUSSION THREAD

**Discussion by Zeatles** [indomus/forum#900] [HUB: Home Assistant]
Ciao a tutti.
Sto integrando il mio impianto domotico con hassio tramite un software che agisca da gateway tra i due come client mqtt.
Ho un problema relativo alle luci (configurate come switch mqtt in hassio) in quando volendo monitorare su hassio la pressione fisica degli interruttori, vado ad aggiornare in tempo reale (circa ogni 500ms) lo stato con un messaggio mqtt. Questo peró mi crea uno strano effetto sullo switch su hassio che solo nel caso di luce accesa ogni tot secondi passa allo stato Off per poi tornare subito a On.
Mi sapete dire dove potrebbere essere il problema?
Grazie mille.

**Comment by snakuzzo** [comment]
No se non condividi i dettagli.
Fai vedere come hai integrato il tutto.
A pelle mi verrebbe da dire che è un tantinello esagerato quello che stai facendo. Pubblicare in continuazione ogni 500ms mi sembra troppo, ma immagino tu non abbia alternative...o sbaglio?
Ad ogni modo condividi il tutto e vediamo
